examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -739,13 +739,22 @@ void WindowManager::UpdateLCD()
         Attributes::CurrentPositionTilt::Get(cover.mEndpoint, tilt);
         chip::DeviceLayer::PlatformMgr().UnlockChipStack();
 
-#ifdef DISPLAY_ENABLED
         if (!tilt.IsNull() && !lift.IsNull())
         {
             LcdPainter::Paint(slLCD, type, lift.Value(), tilt.Value(), mIcon);
         }
-#endif
     }
+#ifdef QR_CODE_ENABLED
+    else
+    {
+        chip::MutableCharSpan qrCode(mQRCodeBuffer);
```

Comment:
This seems like duplication. Since the Window-app was migrated to use the BaseApplication all of the content present in this PR should theoritically be already present in the BaseApplication..... Last time I've check everything was working fine with the Window-app Thread.... Why is this needed at all ?

Reply:
Jf is right here. the base application is the one dealing with the QR code. See Function
`BaseApplication::OutputQrCode(bool refreshLCD)` and its use.

Reply:
Noticed that this part of code is not required for displaying QR. Will remove this.